### ISSUE TITLE
Which 32 classes in ADE20k is used to train deeplabv3plus_mobilenet_edgetpuv2?

### ISSUE THREAD

**Issue by nnami21** [tensorflow/models#10863]
https://github.com/tensorflow/models/tree/master/official/projects/edgetpu/vision/READme.md

I am trying to generate mIOU for deeplabv3plus_mobilenet_edgetpuv2_m on ADE20k 32 classes dataset, however, the [official segmentation mask for ADE20k](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip) contains 150 classes. Please provide a text file containing the 32 ADE20k classes used to train deeplabv3plus_mobilenet_edgetpuv2_m 

I have tried to locate this information in the official repository, however, ADE20k segmentations masks are downloaded from the gs server I don't have access too

**Comment by laxmareddyp** [comment]
Hi @fyangf

Can you please look into this issue when you have time.

Thanks

**Comment by fyangf** [comment]
@laxmareddyp This is owned by edgetpu team. We need to reach out to them to get this info.

**Comment by malla456** [comment]
Hi  @sultanchamberlain ,
As this issue has been inactive for a while, we are following up. If you managed to get the 32-class dataset you needed, please close this issue. If you are still blocked by this, kindly open a new issue with your current status and latest findings.
Thank you .
